Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -5,16 +5,20 @@ function Add-Tls12InSession {
     param()
 
     try {
-        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
-            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
+        if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -notcontains 'Tls12') {
```

Comment:
Should we use the HasFlag() method of Enum class? The if condition would look like if ( -Not [System.Net.ServicePointManager]::SecurityProtocol.HasFlag([System.Net.SecurityProtocolType]::Tls12) )

Refer: https://msdn.microsoft.com/en-us/library/system.enum.hasflag(v=vs.110).aspx

Reply:
+1

Reply:
hasFlag is available since .Net 4.0 only. Let's not take that dependency.